qp_klp/Protocol.py

```diff
@@ -0,0 +1,295 @@
+from sequence_processing_pipeline.ConvertJob import ConvertJob
+from sequence_processing_pipeline.TellReadJob import TellReadJob
+from sequence_processing_pipeline.SeqCountsJob import SeqCountsJob
+from sequence_processing_pipeline.TRIntegrateJob import TRIntegrateJob
+from sequence_processing_pipeline.PipelineError import PipelineError
+from os.path import join, split
+from re import match
+from os import makedirs, rename, walk
+from metapool import load_sample_sheet
+
+
+PROTOCOL_NAME_NONE = "None"
```

Comment:
This is never used again, except in line 24; why not just define there?

Reply:
I'm taking a page out of metapool in that I'm starting to define global string constants at the top of the appropriate file. I actually have 'ASSAY_NAME_NONE' defined for Assays as well. It's true that it's currently only used in one case, but IMHO it's good to define it with the others so that they're all defined in the same place rather than, 'two cases up here and the one negative case down there where you're likely to forget about it.'